src/squlearn/kernel/ml/qsvc.py

```diff
@@ -60,9 +66,16 @@ class QSVC(SVC):
     --------
     """
 
-    def __init__(self, *, quantum_kernel: KernelMatrixBase, **kwargs) -> None:
+    def __init__(
+            self,
+            *,
+            quantum_kernel: KernelMatrixBase,
+            regularization: Union[str, None] = None,
+            **kwargs) -> None:
         self.quantum_kernel = quantum_kernel
-        super().__init__(kernel=self.quantum_kernel.evaluate, **kwargs)
+        super().__init__(
+            kernel=lambda x, y: self.quantum_kernel.evaluate(x, y,
+            regularization=regularization), **kwargs)
```

Comment:
too hacky?

Reply:
I don't think so. PEP8 favors `def`` over `lambda` but I like it 😄